server/app/views/applicant/ApplicantLayout.java

```diff
@@ -315,7 +316,8 @@ private DivTag maybeRenderTiButton(
                       "opacity-75",
                       StyleUtils.hover("opacity-100"),
                       ButtonStyles.SOLID_BLUE_TEXT_XL))
-          .with(
+          .condWith(
+              !request.uri().contains("admin/tiDash"),
```

Comment:
I recognize that using the string literal `admin/tiDash` here isn't ideal. Calling `controllers.ti.routes.TrustedIntermediaryController.dashboard(...).url()` isn't possible because it adds some query params to the URL which cannot be compared to for equality with `request.uri()`.

Open to alternate suggestions.

Reply:
Can we strip off the query params before making the comparison? Maybe by dropping it in a `Uri` and pulling out the path to compare?

Reply:
Done!